alex's tasks

```diff
@@ -0,0 +1,59 @@
+1 Run plain ruby code
+with irb
+with ruby command + .rb file
+use require and require_relative to load code from another file
+
+2 Know where the official ruby documentation is and actually use it
+ruby-doc.org
+
+3 Built-in types
+Strings
+Symbols
+Arrays and Hashes
+Add elements
+Remove elements
+Access elements
+Iterate through elements
+Handful of methods from Enumerable
+#all?
+#any?
+#collect` / `#map`
+#each_with_index and #each_with_object
+#inject / #reduce
+#select and #reject
+#uniq
+implement <=> operator for #min #max #sort
+Able to determine whether an object is able to respond to a specific method
+
+4 Simple methods (actually functions: not within a class)
+Able to define your own methods
```

Comment:
Methods - add behaviour that is useful to have for particular type of object
.to_a and .class are methods defined on all objects